tmp/pids/server.pid

```diff
@@ -0,0 +1 @@
+20406
```

Comment:
Add the .gitignore to this folder, you can use this gitignore as reference https://github.com/github/gitignore/blob/master/Rails.gitignore